splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -502,6 +502,7 @@ def generate(
             logger.error(f"globalConfig file is not valid. Error: {e}")
             sys.exit(1)
         global_config.update_addon_version(addon_version)
+        global_config.add_ucc_version(__version__)
```

Comment:
This file will change the globalConfig not only in `output/.../appserver/...` folder but also in the local repository itself?

Reply:
It does - the same way as it alters the add-on version in local repository.

Reply:
I mean that since the user can affects its original source globalconfig simply adding the --ta-version parameter this should work the same way (more robust though since user cannot unknowingly change it)

Reply:
@artemrys Are you ok with this approach?

Reply:
I think it is okay, I pushed one small refactoring.